rerun_py/src/arrow.rs

```diff
@@ -50,6 +50,8 @@ pub fn build_data_row_from_components(
     components: &PyDict,
     time_point: &TimePoint,
 ) -> PyResult<DataRow> {
+    let row_id = RowId::new(); // Create row-id as early as possible. It has a timestamp and is used to estimate e2e latency.
```

Comment:
similar story and solution proposal as in C++, but here I don't have much data on where all the time goes